engine-standalone-storage/src/lib.rs

```diff
@@ -37,6 +37,21 @@ pub enum StoragePrefix {
     EngineAccountId = 0x07,
 }
 
+impl From<StoragePrefix> for u8 {
```

Comment:
Could you let me know why you added this implementation? The annotation `#[repr(u8)]` allows you to convert any variant of `StoragePrefix` into `u8` type. 

```rust
let block_hash: u8 = StoragePrefix::BlockHash as u8;
```